Makefile

```diff
@@ -15,6 +15,17 @@ pc-dangerous.img: ubuntu-core-desktop-22-amd64-dangerous.model $(EXTRA_SNAPS)
 	  $(foreach snap,$(ALL_SNAPS),--snap $(snap)) $<
 	mv dangerous/pc.img pc-dangerous.img
 
+pi.img: ubuntu-core-desktop-22-pi.model $(EXTRA_SNAPS)
+	rm -rf dangerous/
+	ubuntu-image snap --output-dir img --image-size 12G \
```

Comment:
Note the PC images are now 20G, it 12G still appropriate for Pi?

Reply:
Yes, I think so.  For the Pi we can assume they will be written to an SD card directly.  The pc.img could be booted in a VM directly, and at 12G had only a couple hundred megs free.